docs/source/common/test_set.rst

```diff
@@ -144,5 +144,12 @@ Apart from this ``.validate`` has same API as ``.test``, but would rely respecti
     ``.validate`` method uses the same validation logic being used under validation happening within
     :meth:`~pytorch_lightning.trainer.trainer.Trainer.fit` call.
 
+.. warning::
+
+    When using ``trainer.validate()``, it is recommended to use a single device since Distributed Training such as DDP internally
```

Comment:
```suggestion
    When using ``trainer.validate()``, it is recommended to use a single device since distributed strategies such as DDP internally
```